packages/costa/src/types.ts

```diff
@@ -1,30 +1,60 @@
 import { ParsedUrlQuery } from 'querystring';
 
 /**
- * enum of script type
+ * There are three types of Pipcook script: `DataSource`, `Dataflow`, `Model`.
  */
-export enum ScriptType { DataSource, Dataflow, Model }
+export enum ScriptType {
+  /**
+   * The `DataSource` script needs to collect all the origin data,
+   * and offers an API object for data access.
+   */
+  DataSource,
```

Comment:
DataSource/DataFlow vs. Datasource/Dataflow?

Reply:
Dataflow could be one word; while `DataSource` usually refers to a combination of two words.